test/plug/static_test.exs

```diff
@@ -171,8 +171,9 @@ defmodule Plug.StaticTest do
 
     plug Plug.Static,
       at: "/",
-      from: Path.expand("..", __DIR__),
-      only: ~w(fixtures test_helper.exs)
+      from: Path.expand("../fixtures", __DIR__),
+      only: ~w(ssl static.txt),
+      prefix_only: ~w(file)
```

Comment:
Then I need two plug modules because the original plug already has test_helper.exs in only.